aspnetcore/grpc/performance.md

```diff
@@ -191,7 +191,39 @@ For more information about how flow control works, see [HTTP/2 Flow Control (blo
 > [!IMPORTANT]
 > Increasing Kestrel's window size allows Kestrel to buffer more data on behalf of the app, which possibly increases memory usage. Avoid configuring an unnecessarily large window size.
 
-## Streaming
+## Gracefully complete streaming calls
+
+Try to complete streaming calls gracefully. Gracefully completing calls avoids unnecessary errors and allow servers to reuse internal data structures between requests.
```

Comment:
```suggestion
Ensure streaming calls are completed gracefully to prevent errors and enable server-side reuse of internal data structures. 
```

Reply:
Just a suggestion but not necessary.

Reply:
Ensure is too strong a word. They don't have to gracefully complete.